background.html

```diff
@@ -626,39 +623,42 @@ <h3 class='name' dir='auto'> {{ title }} </h3>
     {{/action }}
   </script>
 
-  <script type='text/x-tmpl-mustache' id='install_flow_template'>
-    <div id='step1' class='step'>
+  <script type='text/x-tmpl-mustache' id='install-choice'>
+    <div class='step'>
       <div class='inner'>
         <div class='step-body'>
           <img id='signal-icon' src='images/icon_250.png'/>
           <h1>{{ installWelcome }}</h1>
           <p>{{ installTagline }}</p>
         </div>
         <div class='nav'>
-          <div> <a class='button step2'>{{ installGetStartedButton }}</a> </div>
-          <span class='dot step1 selected'></span>
-          <span class='dot step2'></span>
-          <span class='dot step3'></span>
+          <div> <a class='button new'>{{ installNew }}</a> </div>
+          <div> <a class='button import'>{{ installImport }}</a> </div>
         </div>
       </div>
     </div>
+  </script>
 
-    <div id='step2' class='step'>
+  <script type='text/x-tmpl-mustache' id='install_flow_template'>
+
+    <div id='step2' class='step hidden'>
       <div class='inner'>
         <div class='step-body'>
           <img id='signal-phone' src='images/signal-phone.png'>
           <p>{{{ installSignalLink }}}</p>
         </div>
         <div class='nav'>
           <div> <a class='button step3'>{{ installIHaveSignalButton }}</a> </div>
-          <span class='dot step1'></span>
-          <span class='dot step2 selected'></span>
-          <span class='dot step3'></span>
+          <div class='dot-container'>
+            <span class='dot step1'></span>
+            <span class='dot step2 selected'></span>
```

Comment:
One thing you an do with that old onboarding system is go back and forward among the steps. We don't really want that, and there are only two steps anyway... Is part of it that you want to move to the blue background?